libs/functions/generate_api.py

```diff
@@ -156,9 +160,19 @@ def generate_region_timeseries(
             for row in model_output.data.to_dict(orient="records")
         ]
 
+    metrics_timeseries = []
+    region_level_metrics = calculate_top_level_metrics_for_timeseries(
+        timeseries, region_summary
+    ).to_dict(orient="records")
+    for metric_row in region_level_metrics:
+        MetricsTimeseriesRow(**metric_row, date=row[CommonFields.DATE])
```

Comment:
don't think this is actually getting added to the timeseries :) 

I think it would be helpful to add a quick test making sure that the metrics data is being added to the output - 
[here](https://github.com/covid-projections/covid-data-model/blob/master/test/libs/generate_api_test.py) is where the existing tests are

Reply:
Doh, I think it got removed in a previous change. I will that test.